lib/public/script.js

```diff
@@ -56,7 +56,7 @@ function deepLink(options) {
     document.location = url;
     setTimeout(function () {
       document.location = fallback;
-    }, 250);
+    }, 2500);
```

Comment:
Any reason to increas it ?
2.5 sec is noticable

Reply:
I am facing the same issue as [#27](https://github.com/mderazon/node-deeplink/pull/27#issuecomment-637197647).
in my case, app usually takes approx 1.5 sec to load.
When I open the app, it immediately redirects me to the app store. so I add some time to the app gets load perfectly